pkgs/top-level/all-packages.nix

```diff
@@ -19078,6 +19076,8 @@ with pkgs;
 
   martyr = callPackage ../development/libraries/martyr { };
 
+  materia-theme = callPackage ../misc/themes/materia-theme { };
```

Comment:
Is this worth an entry in `pkgs/top-level/aliases.nix` or would consider it a new project now?

Reply:
Probably no package depends on Flat-Plat. So the renaming would not have effect in other packages. I think the aliases is not worth in this case. Users of Flat-Plat would rename it in the configuration. Maybe a comment in file `all-packages.nix` or a mention of the old name in the description of the new package would help users.